### DISCUSSION TITLE
使用 rspack ，是否目前是否可以通过builtin:swc-loader 进行 swc 插件扩充？

### DISCUSSION THREAD

**Discussion by thinkeyy** [web-infra-dev/modern.js#4447] [Q&A]
https://github.com/web-infra-dev/rspack/issues/3067

**Comment by chenjiahan** [comment]
目前还不支持，参考下面的回复：

![image](https://github.com/web-infra-dev/modern.js/assets/7237365/38eaf34c-ff75-4020-89d2-709a3983717e)

**Comment by thinkeyy** [comment]
Thank you for your reply ，
I plan to migrate some old projects to rspack,In these old projects I used a custom babel plugin to help configure cssmodule automatically, the babel plugin transform
import a from 'foo.less';
Import 'foo.less';
import c from 'foo.sass';
import d from '../foo.css';
to 
import a from "foo.less?modules";
Import 'foo.less';
import c from "foo.sass?modules";
import d from "../foo.css?modules";
Then , with webpack rule.resourceQuery(/modules/)  open cssmodule。
Now , use repack， Do you have any suggestions for this.

**Reply by chenjiahan** [reply]
Sorry I have no suggestion for this feature, you can continue to discuss it in https://github.com/web-infra-dev/rspack/issues/3067
